services-turf/src/main/java/com/mapbox/turf/TurfConversion.java

```diff
@@ -163,4 +175,58 @@ public static double convertLength(@FloatRange(from = 0) double distance,
     }
     return radiansToLength(lengthToRadians(distance, originalUnit), finalUnit);
   }
+
+  /**
+   * Combines a FeatureCollection of {@link Point} , {@link LineString},
+   * or {@link Polygon} features into {@link MultiPoint}, {@link MultiLineString},
+   * or {@link MultiPolygon}.
+   *
+   * @param featureCollection a {@link FeatureCollection} that has {@link Feature}
+   *                          objects which are all {@link Point}, {@link LineString},
+   *                          or {@link Polygon} geometries.
+   * @return a Geometry which can be used to cast {@link MultiPoint},
+   * {@link MultiLineString}, or {@link MultiPolygon}
+   * @since 4.7.0
+   */
+  public static Geometry combine(@NonNull FeatureCollection featureCollection) {
```

Comment:
I think the code above won't work because `LineString` can't be cast to a `Point`          (`pointList.add((G)singleFeature.geometry());`) and other similar situations.

Reply:
http://turfjs.org/docs/#combine

> Returns FeatureCollection <(MultiPoint|MultiLineString|MultiPolygon)> - a FeatureCollection of corresponding type to input

Return type should be a FeatureCollection